implementations/rust/ockam/ockam_rust_elixir_nifs/README.md

```diff
@@ -0,0 +1,56 @@
+# ockam_rust_elixir_nifs
+
+[![crate][crate-image]][crate-link]
+[![docs][docs-image]][docs-link]
+[![license][license-image]][license-link]
+[![discuss][discuss-image]][discuss-link]
+
+Ockam is a library for building devices that communicate securely, privately
+and trustfully with cloud services and other devices.
+
+This library builds a NIF for our Elixir library
+To build the NIF module:
+
+- Your NIF will now build along with your project.
+
+### To load the NIF:
+
+```elixir
+defmodule Ockly do
```

Comment:
We should change the name on elixir side as well,  instead of letting it Ockly.   But that can be done latter

Reply:
I'll open a follow up PR